docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,120 @@
+|{ecs-ref}/ecs-package.html[`package.*`]|`package.*`|ECS `package.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-process.html[`process.*`]| `process.*`| ECS `process.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-registry.html[`registry.*`] | `registry.*`| ECS `registry.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-related.html[`related.*`] |`related.*` |ECS `related.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-rule.html[`rule.*`]| `rule.*`| ECS `rule.*` fields copied from the source document, if present, for custom query and indicator match rules.
+
+NOTE: These fields are not related to the detection rule that generated the alert.
+
+|{ecs-ref}/ecs-server.html[`server.*`] |`server.*`|ECS `server.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-service.html[`service.*`]|`service.*`|ECS `service.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-source.html[`source.*`] | `source.*` | ECS `source.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-tracing.html#field-span-id[`span.*`] |`span.*`|ECS `span.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-threat.html[`threat.*`]| `threat.*`|ECS `threat.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-tls.html[`tls.*`] | `tls.*`| ECS `tls.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-tracing.html[`trace.*`]| `trace.*`|ECS `trace.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-tracing.html#field-transaction-id[`transaction.*`]|`transaction.*`| ECS `transaction.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-url.html[`url.*`]|`url.*`|ECS `url.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-user.html[`user.*`]|`user.*`| ECS `user.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-user_agent.html[`user_agent.*`]|`user_agent.*`| ECS `user_agent.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-vulnerability.html[`vulnerability.*`]|`vulnerability.*`|ECS `vulnerability.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|`signal.ancestors.*`|`kibana.alert.ancestors.*`|
```

Comment:
Move all of these signal rules up in the table so they're in alphabetical order with the rest of the list.